appstore-metadata-service/src/test/java/com/lgi/appstore/metadata/api/delegate/TestContainerPostgresSQLDelegate.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2023 Liberty Global Technology Services BV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.lgi.appstore.metadata.api.delegate;
+
+import org.testcontainers.containers.ContainerState;
+import org.testcontainers.delegate.AbstractDatabaseDelegate;
+import org.testcontainers.exception.ConnectionCreationException;
+import org.testcontainers.ext.ScriptUtils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.util.Properties;
+
+public class TestContainerPostgresSQLDelegate extends AbstractDatabaseDelegate<Connection> {
+    private final static String USER = "user";
```

Comment:
Perhaps we could rename it to `USER_PROPERTY_KEY`.  At first glance it looks like a login and password for PSQL.

Reply:
Changed 👍